packages/front-end/components/Features/ScheduleInputs.tsx

```diff
@@ -3,54 +3,35 @@ import clsx from "clsx";
 import { format } from "date-fns";
 import { format as formatTimeZone } from "date-fns-tz";
 import React, { useEffect, useState } from "react";
+import { UseFormReturn } from "react-hook-form";
+import { useUser } from "@/services/UserContext";
 import Field from "../Forms/Field";
 import SelectField from "../Forms/SelectField";
+import PremiumTooltip from "../Marketing/PremiumTooltip";
+import UpgradeMessage from "../Marketing/UpgradeMessage";
+import Toggle from "../Forms/Toggle";
 import styles from "./ScheduleInputs.module.scss";
 
 interface Props {
   defaultValue: ScheduleRule[];
   onChange: (value: ScheduleRule[]) => void;
+  setShowUpgradeModal: (value: boolean) => void;
+  // eslint-disable-next-line
+  form: UseFormReturn<any>;
```

Comment:
nit: it would be good to attempt to type the return value instead of using <any> if possible. Sometimes it's more trouble than it's worth, but you end up sacrificing clarity and safety, which I think is important given all the weird overrides going on. If it's not really doable though without significant effort though, <any> is fine.